### ISSUE TITLE
Filter on examiner in marking view - part two

### ISSUE THREAD

**Issue by a16seplu** [HGustavs/LenaSYS#4339]
We need an interactive looking glass filter in the marking view

Implemented "for free" by using sortable table

If we write the name of an examiner in this box only students examined by this teacher are shown.

**This filter is remembered if one returns to that page later on.**

**Comment by a17pongo** [comment]
@HGustavs by marking view, do you mean resulted page? 
and might this be fixed by implementing the boolean search?
There seems to be similar issues related to filtering the resulted view which could have been solved.

**Comment by b17nicsv** [comment]
@HGustavs Bump

**Comment by HGustavs** [comment]
Yes in resulted, filtering on examiner... this should already work in this way if search works?

**Comment by HGustavs** [comment]
Sortable table?

**Comment by a18matna** [comment]
![filteronexaminator](https://user-images.githubusercontent.com/49142171/80074346-4bdb6e00-8549-11ea-9924-036e1a8b577e.png)

**Comment by a18matna** [comment]
Filtering on examinator Already works, I suggest closing Issue
